Misc/NEWS.d/next/Tests/2019-10-02-12-27-58.bpo-38347.zdkCBX.rst

```diff
@@ -0,0 +1 @@
+Add a test case for the recursive search including filenames with a '-'.
```

Comment:
Please remove this NEWS entry. The other one is enough.

Reply:
Done in 14f5aa0d351fddae1e798b5838f498c648bbcbda

Reply:
Just as a comment: This causes one of the automatic checks to fail again, but I guess this is not a real blocker.